kafka-connect-aws-s3/src/it/scala/io/lenses/streamreactor/connect/aws/s3/source/S3SourceTaskXmlReaderTest.scala

```diff
@@ -0,0 +1,71 @@
+package io.lenses.streamreactor.connect.aws.s3.source
+
+import cats.implicits._
+import io.lenses.streamreactor.connect.aws.s3.config.AuthMode
+import io.lenses.streamreactor.connect.aws.s3.config.S3ConfigSettings._
+import io.lenses.streamreactor.connect.aws.s3.utils.S3ProxyContainerTest
+import org.apache.kafka.connect.source.SourceRecord
+import org.scalatest.concurrent.Eventually.eventually
+import org.scalatest.flatspec.AnyFlatSpecLike
+import org.scalatest.matchers.should.Matchers
+
+import java.io.File
+import scala.jdk.CollectionConverters.ListHasAsScala
+import scala.jdk.CollectionConverters.MapHasAsJava
+
+class S3SourceTaskXmlReaderTest extends S3ProxyContainerTest with AnyFlatSpecLike with Matchers {
+
+  def DefaultProps: Map[String, String] = Map(
+    AWS_ACCESS_KEY                          -> Identity,
+    AWS_SECRET_KEY                          -> Credential,
+    AWS_REGION                              -> "eu-west-1",
+    AUTH_MODE                               -> AuthMode.Credentials.toString,
+    CUSTOM_ENDPOINT                         -> uri(),
+    ENABLE_VIRTUAL_HOST_BUCKETS             -> "true",
+    TASK_INDEX                              -> "1:1",
```

Comment:
This is not a valid value no? First one is the max tasks and the second is the task id, and taskid should be MaxTax-1 (it is zero bound)